code/iaas/cluster-logic/src/main/java/io/cattle/iaas/cluster/process/lock/CertificateAuthorityLock.java

```diff
@@ -0,0 +1,11 @@
+package io.cattle.iaas.cluster.process.lock;
+
+import io.cattle.platform.lock.definition.AbstractBlockingLockDefintion;
+
+public class CertificateAuthorityLock extends AbstractBlockingLockDefintion {
+
+    public CertificateAuthorityLock(Long accountId) {
```

Comment:
Minor, but can we change the argument type to use primitive long instead? So null values don't slip in